modes/helm/evil-collection-helm.el

```diff
@@ -74,7 +74,10 @@
         (overlay-put ov 'window (selected-window))
         (overlay-put ov 'face (let ((bg-color (face-background 'default nil)))
                                 `(:background ,bg-color :foreground ,bg-color)))
-        (setq-local cursor-type nil)))))
+        (setq-local cursor-type nil
+                    evil-emacs-state-cursor '(nil +evil-emacs-cursor-fn)
+                    evil-normal-state-cursor '(nil +evil-default-cursor-fn)
+                    evil-insert-state-cursor '(nil +evil-default-cursor-fn))))))
```

Comment:
I suspect `+evil-emacs-cursor-fn` and `+evil-default-cursor-fn` are functions from doom-emacs.

https://github.com/doomemacs/doomemacs/blob/master/modules/editor/evil/config.el#L101-L104